### ISSUE TITLE
[bug] Missing input digest in readme example for container workflow

### ISSUE THREAD

**Issue by jobroe10** [slsa-framework/slsa-github-generator#543]
**Describe the bug**
While using the recently released generator_container_slsa3.yml and following the exemplary workflow provided in the README, the error message "Input digest is required, but not provided while calling." is thrown.
Looking at the input defined in the example only the following are defined as input parameters:

```
with:
    image: ${{ needs.build.outputs.tag }}
    registry-username: ${{ github.actor }}
    # TODO(https://github.com/slsa-framework/slsa-github-generator/issues/492): Remove after GA release.
    compile-generator: true
```

**To Reproduce**
Using the example workflow described [here](https://github.com/slsa-framework/slsa-github-generator/tree/main/internal/builders/container).

**Expected behavior**
No error is thrown that the input digest is required.

Thank you (:
(probably @ianlewis)

**Comment by laurentsimon** [comment]
Appreciate the issue. This workflow is WIP and not officially supported. We will get to it in the next few weeks.
The tag, for example is not immutable so we're going to be updating the workflow interface to use a full `@sha256:xxx`, for example.

**Comment by Root-DE** [comment]
Also solved here #606
